### PR TITLE
Pirates

### PR DESCRIPTION
**Pull request by MadmanMartian**
some freaking pirates faction that's not working right now.

Thought it'd be relevant, considering the whole objective thing. The objective is always automatically completed, it just shows what "score" the pirates have, unless of course the chest were to be lost or destroyed.

# todo

A lot

### CODE REVIEW

File: code/datums/gamemode/role/pirate.dm [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+	.=..()
+	if(!.)
+		return
+	equip_pirate(antag.current)
+
+/datum/role/pirate/proc/equip_pirate(var/mob/living/carbon/human/H)
+	if (!istype(H))
+		return
+
+	qdel(H.wear_suit)
```
Comment (jknpj) [inline code/datums/gamemode/role/pirate.dm:16]: YEAT ANOTHER REMINDER ABOUT OUTFIT DATUMS

Comment (MadmanMartian) [reply code/datums/gamemode/role/pirate.dm:16]: YEAT

Comment (jknpj) [reply code/datums/gamemode/role/pirate.dm:16]: YEEET

File: code/datums/gamemode/dynamic/dynamic_rulesets_midround.dm [hunk 2]
```diff
@@ -277,6 +277,28 @@
 		return 0
 
 /datum/dynamic_ruleset/midround/from_ghosts/weeaboo/ready(var/forced = 0)
+	assigned += new_character
```
Comment (MadmanMartian) [inline code/datums/gamemode/dynamic/dynamic_rulesets_midround.dm]: fucking H O W

**Pull request closed without merging**